edurange_refactored/user/views.py

```diff
@@ -465,6 +465,11 @@ def scenariosInfo(sId):
 
     players = db_ses.query(User.username).filter(GroupUsers.group_id == StudentGroups.id, StudentGroups.id == gid, GroupUsers.user_id == User.id).all()
 
+    tmp = []
+    for user in players:
+        tmp.append({'username': ''.join(c for c in user.username if c.isalnum())})
+    players = tmp
+
```

Comment:
One potential issue in this change that I can see is that `players` used to be a list of Row objects, where as here it is a list of dictionaries. I seemed to be unable to modify the values stored in the list of row objects returned from the db query and was unsure how to recreate the same type. It seems to work fine like this, but I think there is a more correct way to send this list of players to the webpage.